src/_includes/initiatives.html

```diff
@@ -11,110 +10,119 @@
+          <picture class="col-12 col-md-2 col-lg-2 d-flex justify-content-md-end align-self-md-start justify-content-center">
+            <img class="pt-2 pb-4 pb-md-0"
+                 src="{{ initiative.image.src }}"
+                 alt="{{ initiative.image.alt }}"
+                 width="{{ initiative.image.width }}" />
+          </picture>
+          <div class="col-12 col-md-8 col-lg-8 mb-28 mb-md-40">
+            <h3 class="h2 text-white text-center text-md-start pb-2 pb-md-4 pt-2 pt-md-0">{{ initiative.headline }}</h3>
+            {% for paragraph in initiative.paragraphs %}<p class="text-white">{{ paragraph }}</p>{% endfor %}
+            <div class="d-grid d-md-block rounded-0 border-bottom border-white border-2">
+              <a href="{{ initiative.button.url }}"
+                 class="btn btn-outline-light fw-bolder mt-2 mb-28 mb-md-40 {{ initiative.button.class }}">{{ initiative.button.text }}</a>
+            </div>
           </div>
         </div>
-      </div>
-      <div class="row px-2 px-md-0">
-        <div class="col-12 offset-md-2 col-md-8 col-lg-8 mb-28 mb-md-40">
-          <h4 class="small-caps text-white mb-3">Recent News</h4>
-          <div class="tab-list pb-2">
-            {% assign all_press = site.press | reverse %} {% for press in all_press %} {% assign url = press.url %} {% if
-            press.external %} {% assign url = press.external %} {% endif %} {% if press.tags contains initiative.tag %}
-            <article class="press-item">
-              <a
-                class="text-white fw-bold py-2 d-block"
-                rel="noreferrer"
-                href="{{ url }}"
-                {%
-                if
-                press.external
-                %}target="_blank"
-                {%
-                endif
-                %}
-                >{{ press.title }}</a
-              >
-            </article>
-            {% endif %} {% endfor %}
+        <div class="row px-2 px-md-0">
+          <div class="col-12 offset-md-2 col-md-8 col-lg-8 mb-28 mb-md-40">
+            <h4 class="small-caps text-white mb-3">Recent News</h4>
+            <div class="tab-list pb-2">
+              {% assign all_press = site.press | reverse %}
+              {% for press in all_press %}
+                {% assign url = press.url %}
+                {% if
+                  press.external %}
+                  {% assign url = press.external %}
+                {% endif %}
+
+                {% if press.tags contains initiative.tag %}
+                  <article class="press-item">
+                    <a
+                      class="text-white fw-bold py-2 d-block"
+                      rel="noreferrer"
+                      href="{{ url }}"
+                      {%
+                      if
+                      press.external
+                      %}target="_blank"
+                      {%
+                      endif
+                      %}
```

Comment:
I kind of wish this wasn't so stretched out across lines, but I guess it's ok. Better stretched out than squished together